docs/rules/jsx-sort-default-props.md

```diff
@@ -0,0 +1,97 @@
+# Enforce propTypes declarations alphabetical sorting (react/jsx-sort-default-props)
+
+Some developers prefer to sort propTypes declarations alphabetically to be able to find necessary declaration easier at the later time. Others feel that it adds complexity and becomes burden to maintain.
```

Comment:
let's make sure all `propTypes` and `defaultProps` mentions are surrounded by backticks.

Also, "to be able to find necessary declarations easier at a later time" and "becomes a burden to maintain"

Reply:
Done! Should I fix these typos in `sort-prop-types` and `jsx-sort-props` documentation too? Their introduction are identical.